src/luarocks/cmd/write_rockspec.lua

```diff
@@ -67,6 +67,7 @@ rockspec, and is not guaranteed to be complete or correct. ]], util.see_also())
 
    cmd:argument("name", "Name of the rock.")
       :args("?")
+      :action(util.namespaced_name_action)
```

Comment:
Does write_rockspec even need to accept a namespace?

Reply:
I don't think so, I think that was just blanket-added to every command that used a `name`...